packages/docs/pages/reference/elements/algorithm.mdx

```diff
@@ -8,12 +8,30 @@ import {
 
 # `<algorithm>`
 
-Short description of what the `<algorithm>` tag does.
+The `<algorithm>` tag creates a container for the statement of an algorithm
+with optional `title`, `creator`, and `proof`. The structure of the `algorithm`
+tag is completely analogous to that of the following tags:
+[`claim`](./claim), [`corollary`](./corollary),
+[`fact`](./fact), [`identity`](./identity), [`lemma`](./lemma),
+[`proposition`](./proposition), and [`theorem`](./theorem).
```

Comment:
These are all the theorem-like tags, right? I think we should not explicitly list them since more may be added in the future. Perhaps we should limit it to something like
```
`<algorithm>` is a [`<theorem>`](theorem)-like tag
and so shares the attributes and structure with other theorem-like tags.
```
then in the `<theorem>` tag itself, we can be a little more verbose.